source/style-guide/writing/clarify-modifiers.txt

```diff
@@ -91,7 +91,7 @@ Dangling Modifiers
 Modifier clauses at the beginning or end of a sentence should modify
 the subject of the main clause. When the subject is missing or the
 clause modifies another object in the sentence, the clause is called a
-:wikipedia:`dangling modifier </Dangling_modifier?oldid=877876557>`.
+:wikipedia:`dangling modifier <Dangling_modifier?oldid=877876557>`.
```

Comment:
This Wikipedia link goes to a specific revision of the Wikipedia article. Is this intentional? If we're linking to Wikipedia in the style guide, do we try to anchor to specific revisions like this?

Reply:
Anchoring to specific revisions ensures that we're linking to an accurate / safe version of the page. Given Wikipedia's community editing, we don't want to risk linking to a version of the page that could contain wrong information or edited to be offensive.

Reply:
I thought this was explained in the style guide somewhere, but wasn't able to locate it. Anyone know where that is?

Wikipedia content is volatile, so some information may change without us knowing about it. The oldid parameter lets you access the version of the content at a given point in time so you can be more certain as to what content currently exists at that link.

Reply:
Thanks for confirming! I just wanted to make sure.